docs/01-app/04-api-reference/05-config/01-next-config-js/viewTransition.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: ViewTransition
+description: Enable ViewTransition API from React in App Router
+version: experimental
+---
+
+`viewTransition` is an experimental feature that a feature flag to enable the new experimental View Transitions API in React. This new API allows you to animate between different views and components in your application.
```

Comment:
This new API allows you to leverage the View Transition browser API to create seamless transitions [...]

Reply:
let's also link to the mdn documentation for the general concept